jobs/JGLOBAL_WAVE_STAT

```diff
@@ -0,0 +1,55 @@
+#! /usr/bin/env bash
+
+source "${HOMEgfs}/ush/preamble.sh"
```

Comment:
Should these be "JGLOBAL" or "JGEFS" ?

Reply:
I just followed whatever was in similar jobs such as [JGLOBAL_WAVE_POST_BNDPNT](https://github.com/NOAA-EMC/global-workflow/blob/develop/jobs/JGLOBAL_WAVE_POST_BNDPNT). Let me know if it needs to be different.

Reply:
That's a @aerorahul @KateFriedman-NOAA question.

Reply:
Will this job only run with GEFS or will it run for GFS and/or GDAS as well? If more than just GEFS, then `JGLOBAL`.